framework/src/modules/chain/chain.js

```diff
@@ -295,4 +318,64 @@ module.exports = class Chain {
 
 		this.logger.info('Cleaned up successfully');
 	}
+
+	_startLoader() {
```

Comment:
Move these actions to their respective modules, don't thick the initialization logic file.

Reply:
It's not initialisation, chain.js will be orchestrating the modules